components/identifier-generator/front/src/feature/tabs/structure/edit/AutoNumberEdit.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {AutoNumber} from '../../../models';
+import {Field, NumberInput} from 'akeneo-design-system';
+import {useTranslate} from '@akeneo-pim-community/shared';
+import {PropertyEditFieldsProps} from '../PropertyEdit';
+
+const AutoNumberEdit: PropertyEditFieldsProps<AutoNumber> = ({selectedProperty, onChange}) => {
+  const translate = useTranslate();
+  const onDigitsMinChange = (value: string) => {
+    selectedProperty.digitsMin = Number(value);
```

Comment:
Not sure we should change a prop value (same for `FreeTextEdit`).
We could send directly a new object to the onChange
```suggestion
    onChange({...selectedProperty, digitsMin: Number(value)});
```